docs/content/union.md

```diff
@@ -155,17 +155,21 @@ as necessary.
 ### Quota Relevant Policies
 
 Some policies rely on quota information. These policies should be used only if
-your upstreams support the respective quota fields.
-
-| Policy     | Required Field |
-|------------|----------------|
-| lfs, eplfs | Free           |
-| mfs, epmfs | Free           |
-| lus, eplus | Used           |
-| lno, eplno | Objects        |
-
-To check if your upstream supports the field, run `rclone about remote: [flags]`
-and see if the required field exists.
+your upstreams support the respective quota fields. To check if your upstream supports the field,
+run `rclone about remote: [flags]` and see if the required field exists.
+
+| Policy     |   Required Field   |
+|------------|--------------------|
+| lfs, eplfs | Free               |
+| mfs, epmfs | Free               |
+| lus, eplus | Used               |
+| lno, eplno | Objects, Used¹     |
+
+¹ Objects are only available for S3-based upstreams. For better compatibility, since
+rclone v1.72.0 there is a fallback to normal listing (the same one performed by `rclone size`).
```

Comment:
Not sure which version I should use here. It's this one correct? Or shall I change it to v1.71.1?